tuxedo/infinitybook/default.nix

```
@@ -0,0 +1,8 @@
+{ lib, ... }: {
+  imports = [
+    ../../common/pc/laptop
+    ../../common/pc/ssd
+  ];
+
+  hardware.tuxedo-drivers.enable = lib.mkDefault true;
```

I believe this option doesn't exist in nixos stable yet.

you can do something like:

```suggestion
  hardware = lib.optional (options.hardware ? tuxedo-drivers) {
    enable = lib.mkDefault true;
  };
```